src/main/java/org/skriptlang/skript/elements/expressions/displays/ExprDisplayTransformationRotation.java

```diff
@@ -0,0 +1,100 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.elements.expressions.displays;
+
+import org.bukkit.entity.Display;
+import org.bukkit.event.Event;
+import org.bukkit.util.Transformation;
+import org.jetbrains.annotations.Nullable;
+import org.joml.Quaternionf;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Display Transformation Rotation")
+@Description("Returns or changes the transformation rotation of <a href='classes.html#display'>displays</a>.")
```

Comment:
```suggestion
@Description({
        "Returns or changes the transformation rotation of <a href='classes.html#display'>displays</a>.",
        "The left rotation is applied first, with the right rotation then being applied based on the rotated axis."
})
```
I realize the wiki says that the right rotation is the "first" rotation, but this is not borne out in my tests with `/data`.